examples/T3_ILI9341_display/T3_ILI9341_display.ino

```diff
@@ -70,7 +70,7 @@ typedef struct{
 } obd_unit;
 
 typedef struct{
-  uint32_t old_value;
+  int32_t old_value;
```

Comment:
One small unrelated fix to ensure that clearing the previous number doesn't break when a negative number was displayed, such as low temperatures.